src/system/BlocksModule/Resources/views/Admin/view.html.twig

```diff
@@ -16,62 +16,64 @@
 
 {% include 'ZikulaBlocksModule:Admin:filter_form.html.twig' %}
 
-<table id="block-list" class="table table-bordered table-striped">
-    <colgroup>
-        <col id="cBlockId" />
-        <col id="cBlockTitle" />
-        <col id="cBlockDescription" />
-        <col id="cBlockModule" />
-        <col id="cBlockType" />
-        <col id="cBlockPositions" />
-        <col id="cBlockLanguage" />
-        <col id="cBlockState" />
-        <col id="cBlockActions" />
-    </colgroup>
-    <thead>
-        <tr>
-            <th id="hBlockId" scope="col"><a href="{{ sort.bid.url }}" title="{{ __('Sort by ID') }}" class="{{ sort.bid.class }}">{{ __('ID') }}</a></th>
-            <th id="hBlockTitle" scope="col"><a href="{{ sort.title.url }}" title="{{ __('Sort by title') }}" class="{{ sort.title.class }}">{{ __('Title') }}</a></th>
-            <th id="hBlockDescription" scope="col">{{ __('Description') }}</th>
-            <th id="hBlockModule" scope="col">{{ __('Module') }}</th>
-            <th id="hBlockType" scope="col"><a href="{{ sort.blocktype.url }}" title="{{ __('Sort by type') }}" class="{{ sort.blocktype.class }}">{{ __('Type') }}</a></th>
-            <th id="hBlockPositions" scope="col">{{ __('Positions') }}</th>
-            <th id="hBlockLanguage" scope="col"><a href="{{ sort.language.url }}" title="{{ __('Sort by language') }}" class="{{ sort.language.class }}">{{ __('Language') }}</a></th>
-            <th id="hBlockState" scope="col"><a href="{{ sort.state.url }}" title="{{ __('Sort by state') }}" class="{{ sort.state.class }}">{{ __('State') }}</a></th>
-            <th id="hBlockActions" scope="col" class="text-right">{{ __('Actions') }}</th>
-        </tr>
-    </thead>
-    <tbody>
-    {% for block in blocks %}
-    <tr>
-        <td headers="hBlockId">{{ block.bid }}</td>
-        <td headers="hBlockTitle">{{ block.title|safeHtml }}</td>
-        <td headers="hBlockDescription">{{ block.description|safeHtml }}</td>
-        <td headers="hBlockModule">{{ block.module.name }}</td>
-        <td headers="hBlockType">{{ block.blocktype }}</td>
-        <td headers="hBlockPositions">
-            {% for position in block.positions %}
-                {% if not positionavailable(position.name) %} <i class="fa fa-exclamation-triangle text-danger tooltips" title="{{ __('Position not available in current default theme.') }}"></i>{% endif %}
-                <a href="{{ path('zikulablocksmodule_placement_edit', {pid:position.pid}) }}">{{ position.name }}</a>{% if not loop.last %}, {% endif %}
+<div class="table-responsive">
```

Comment:
It makes the table responsive, by adding a horizontal scroll bar, if needed, rather than breaking the layout if the table is too wide.

http://getbootstrap.com/css/#tables-responsive